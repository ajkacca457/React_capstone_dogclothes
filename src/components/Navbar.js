import React from 'react';
import Navlogo from '../images/navlogo.png';

const Navbar = () => (
  <>
    <nav className="navbar navbar-expand-lg px-4 position-fixed">
      <img src={Navlogo} alt="tvshowlogo" style={{ width: '35px', height: '35px' }} />
      <h5 className="ml-3 text-white">TVWORLD</h5>
      <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <i className="fas fa-bars text-light" />
      </button>

      <div className="collapse navbar-collapse" id="navbarSupportedContent">
        <ul className="navbar-nav m-auto text-center">
          <li className="nav-item active mr-2">
            <a className="nav-link text-white" href="#home">
              <i className="fas fa-home mr-1" />
              Home
            </a>
          </li>
          <li className="nav-item mr-2">
            <a className="nav-link text-white" href="#home">
              <i className="fas fa-tv mr-1" />
              TV shows
            </a>
          </li>
          <li className="nav-item">
            <a className="nav-link text-white" href="#home">
              <i className="fas fa-info-circle mr-1" />
              About
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </>
);

export default Navbar;
