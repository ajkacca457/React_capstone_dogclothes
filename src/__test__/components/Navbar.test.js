import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import Navbar from '../../components/Navbar';

describe('Navbar Component', () => {
  it('renders correctly', () => {
    const navtree = TestRenderer
      .create(
        /* eslint-disable no-unused-vars */
        <BrowserRouter>
          {' '}
          <Navbar />
          {' '}
        </BrowserRouter>,
      )
      .toJSON();
      /* eslint-enable no-unused-vars */
    expect(navtree).toMatchSnapshot();
  });
});
