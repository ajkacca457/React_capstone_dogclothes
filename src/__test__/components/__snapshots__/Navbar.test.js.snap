// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component renders correctly 1`] = `
Array [
  " ",
  <nav
    className="navbar navbar-expand-lg px-4 position-fixed"
  >
    <img
      alt="tvshowlogo"
      src="navlogo.png"
      style={
        Object {
          "height": "35px",
          "width": "35px",
        }
      }
    />
    <h5
      className="ml-3 text-white"
    >
      TVWORLD
    </h5>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarSupportedContent"
      data-toggle="collapse"
      type="button"
    >
      <i
        className="fas fa-bars text-light"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav m-auto text-center"
      >
        <li
          className="nav-item active mr-2"
        >
          <a
            className="nav-link text-white"
            href="/home"
            onClick={[Function]}
          >
            <i
              className="fas fa-home mr-1"
            />
            Home
          </a>
        </li>
        <li
          className="nav-item mr-2"
        >
          <a
            className="nav-link text-white"
            href="/shows"
            onClick={[Function]}
          >
            <i
              className="fas fa-tv mr-1"
            />
            TV shows
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link text-white"
            href="/about"
            onClick={[Function]}
          >
            <i
              className="fas fa-info-circle mr-1"
            />
            About
          </a>
        </li>
      </ul>
    </div>
  </nav>,
  " ",
]
`;
